configurablecare/app/src/main/java/com/google/fhir/examples/configurablecare/care/TaskManager.kt

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright 2022-2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.fhir.examples.configurablecare.care
+
+import com.google.android.fhir.FhirEngine
+import com.google.android.fhir.search.Operation
+import com.google.android.fhir.search.Search
+import com.google.android.fhir.search.search
+import java.time.Instant
+import java.time.Period
+import java.util.Date
+import org.hl7.fhir.r4.model.CarePlan
+import org.hl7.fhir.r4.model.IdType
+import org.hl7.fhir.r4.model.PractitionerRole
+import org.hl7.fhir.r4.model.Questionnaire
+import org.hl7.fhir.r4.model.Reference
+import org.hl7.fhir.r4.model.ResourceType
+import org.hl7.fhir.r4.model.ServiceRequest
+import org.hl7.fhir.r4.model.Task
+import org.hl7.fhir.r4.model.Task.TaskStatus
+
+class TaskManager(
+  private var fhirEngine: FhirEngine,
+  private val taskConfigMap: MutableMap<String, String>
```

Comment:
Maybe instead of taskConfigMap we pass `requestResourceConfigurations`  from ImplementationGuideConfig class ?

Reply:
Modified to use `requestResourceConfigurations` instead